package.json

```diff
@@ -43,17 +44,18 @@
     "ember-cli-moment-shim": "2.0.0",
     "ember-cli-notifications": "^3.2.0",
     "ember-cli-showdown": "^2.5.0",
-    "ember-cli-sri": "^2.0.0",
+    "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^1.2.0",
     "ember-computed-decorators": "0.2.2",
-    "ember-data": "^2.4.0",
+    "ember-data": "^2.5.0",
+    "ember-disable-prototype-extensions": "^1.1.0",
```

Comment:
We have it listed as a `devDependency` so that the demo app and tests for bunsen run without the prototype extensions. A consumer still has prototype extensions present unless they add `ember-disable-prototype-extensions` to their dependencies.